.vscode/launch.json

```diff
@@ -82,8 +82,8 @@
       "name": "Mocha Tests",
       "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
       "args": [
-        "--opts",
```

Comment:
Are you sure you want this change?

Reply:
Yes. The debbugger wasn't working with this. It was deprecated